services/identity/src/test/java/com/crapi/service/Impl/UserServiceImplTest.java

```diff
@@ -19,23 +19,11 @@
 import com.crapi.config.JwtAuthTokenFilter;
 import com.crapi.config.JwtProvider;
 import com.crapi.constant.UserMessage;
-import com.crapi.entity.ChangeEmailRequest;
-import com.crapi.entity.ProfileVideo;
-import com.crapi.entity.User;
-import com.crapi.entity.UserDetails;
+import com.crapi.entity.*;
 import com.crapi.enums.ERole;
 import com.crapi.exception.EntityNotFoundException;
-import com.crapi.model.CRAPIResponse;
-import com.crapi.model.ChangeEmailForm;
-import com.crapi.model.DashboardResponse;
-import com.crapi.model.JwtResponse;
-import com.crapi.model.LoginForm;
-import com.crapi.model.LoginWithEmailToken;
-import com.crapi.model.SignUpForm;
-import com.crapi.repository.ChangeEmailRepository;
-import com.crapi.repository.ProfileVideoRepository;
-import com.crapi.repository.UserDetailsRepository;
-import com.crapi.repository.UserRepository;
+import com.crapi.model.*;
+import com.crapi.repository.*;
```

Comment:
Can we revert importing .* and add just the required packages across all files. We need to change a setting in VSCode here

Reply:
Done